lib/ansible/modules/system/aix_lvg.py

```diff
@@ -0,0 +1,273 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2017, Kairo Araujo <kairo@kairo.eti.br>
+# GNU General Public License v3.0+ (see COPYING or
+# https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+author: "Kairo Araujo (@kairoaraujo)"
+module: aix_lvg
+short_description: Configure LVM volume groups for AIX
+description:
+  - This module creates, removes or resize volume groups on AIX LVM.
+version_added: 2.5
+options:
+  vg:
+    description:
+    - Volume Group name
+    required: true
+  pvs:
+    description:
+    - List of comma-separated devices to use as physical devices in this volume
+      group. Required when creating or extending ('present' state) the volume
+      group. If not informed reducing ('absent' state) the volume group will be
+      removed.
+    required: false
+  pp_size:
+    description:
+    - Size of the physical partition in megabytes.
+    choices: [1, 2, 4, 8, 16, 32, 64, 128, 256, 512, 1024]
+    default: 32
+    required: false
+  vg_type:
+    description:
+    - Volume Group type.
+    choices: ["normal", "big", "scalable"]
+    default: normal
+    required: false
+  state:
+    description:
+    - Control if the volume group exists.
+    choices: [ "present", "absent" ]
+    default: present
+    required: false
+  force:
+    description:
+    - Forces volume group creation
+    choices: ['yes', 'no']
+    default: no
+notes:
+  - AIX will permit remove VG only if all LV/Filesystems aren't busy
+  - module does not modify PP size for already present volume group
+'''
+
+EXAMPLES = '''
+# Create a volume group datavg
+aix_lvg:
+    vg: datavg
```

Comment:
Standard YAM indentation is two characters. Please change it to maintain the same indentation like in the `DOCUMENTATION` block. The same for all examples.